integration_test/pg/check_constraint_test.exs

```diff
@@ -0,0 +1,88 @@
+defmodule Ecto.Integration.CheckConstraintTest do
+  use ExUnit.Case, async: true
+
+  alias Ecto.Integration.TestRepo
+  import Ecto.Migrator, only: [up: 4, down: 4]
+
+  defmodule TableCreationMigration do
```

Comment:
I'm happy to merge them, but the reason I initially made them separate was that I was afraid that if `down` did both `drop check_constraint...` and `drop table`, it would return `:ok` and appear to have worked (the constraint would be destroyed along with the table) even if `drop check_constraint` didn't work. Not a problem?

BTW I also used a different version timestamp than what I'd seen in other migrations because otherwise I got intermittent test failures, saying a migration was already up. Seemed like a race condition, maybe because of versions + `async: true`.